packages/vite/src/node/index.ts

```diff
@@ -100,6 +100,8 @@ export type {
 } from './server/transformRequest'
 export type { HmrOptions, HmrContext } from './server/hmr'
 
+export type { BindShortcutsOptions, CLIShortcut } from './shortcuts'
```

Comment:
If `BindShortcutsOptions` is exposed (TBV), we can remove the type hack on `_shortcutsOptions`